packages/twenty-server/src/engine/workspace-manager/workspace-sync-metadata/constants/standard-field-ids.ts

```diff
@@ -37,11 +37,13 @@ export const attachmentStandardFieldIds = {
   name: '20202020-87a5-48f8-bbf7-ade388825a57',
   fullPath: '20202020-0d19-453d-8e8d-fbcda8ca3747',
   type: '20202020-a417-49b8-a40b-f6a7874caa0d',
+  storageDriverType: '20202020-a417-49b8-a40b-f6a7874caa0e',
   author: '20202020-6501-4ac5-a4ef-b2f8522ef6cd',
   activity: '20202020-b569-481b-a13f-9b94e47e54fe',
   person: '20202020-0158-4aa2-965c-5cdafe21ffa2',
   company: '20202020-ceab-4a28-b546-73b06b4c08d5',
   opportunity: '20202020-7374-499d-bea3-9354890755b5',
+  message: '20202020-7374-499d-bea3-9354890755b6',
```

Comment:
Honestly I have no idea at all whether this is correct or not. I read the comment on the top of this file and checked the PR that introduced this file, but still pretty much no idea whether I can make up the ID or get it from somewhere 🤷‍♂️